jupyter-docker/jupyter_localize_extension.py

```diff
@@ -0,0 +1,19 @@
+from notebook.utils import url_path_join
+from notebook.base.handlers import IPythonHandler
+
+class LocalizeHandler(IPythonHandler):
+    def post(self):
+        self.set_status(200)
```

Comment:
So I understand: this is where the actual localize code will go?

Reply:
yes

Reply:
https://github.com/DataBiosphere/leonardo/blob/he_localize/jupyter-docker/jupyter_localize_extension.py btw

Reply:
nice